toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -291,20 +338,53 @@ func isPackageInstalled(imageChroot *safechroot.Chroot, packageName string) bool
 	return true
 }
 
-func cleanTdnfCache(imageChroot *safechroot.Chroot) error {
+func cleanTdnfCache(imageChroot *safechroot.Chroot, toolsChroot *safechroot.Chroot) error {
 	logger.Log.Infof("Cleaning up RPM cache")
-	// Run all cleanup tasks inside the chroot environment
-	return imageChroot.UnsafeRun(func() error {
-		tdnfArgs := []string{
-			"-v", "clean", "all",
+
+	tdnfArgs := []string{
+		"-v", "clean", "all",
+	}
+
+	tdnfChroot := imageChroot
+	if toolsChroot != nil {
+
+		err := appendTdnfArgsForToolsChroot(&tdnfArgs)
+		if err != nil {
+			return fmt.Errorf("failed to append tdnf args for tools chroot:\n%w", err)
 		}
+		tdnfChroot = toolsChroot
+	}
+
+	// Run all cleanup tasks inside the chroot environment
+	return tdnfChroot.UnsafeRun(func() error {
 		err := shell.NewExecBuilder("tdnf", tdnfArgs...).
 			LogLevel(logrus.TraceLevel, logrus.DebugLevel).
 			ErrorStderrLines(1).
 			Execute()
 		if err != nil {
-			return fmt.Errorf("Failed to clean tdnf cache: %w", err)
+			return fmt.Errorf("failed to clean tdnf cache:\n%w", err)
 		}
 		return nil
 	})
 }
+
+// Update the TDNF arguments to include the release version and install root options for the tools chroot.
+func appendTdnfArgsForToolsChroot(tdnfArgs *[]string) error {
```

Comment:
`appendTdnfArgsForToolsChroot` should have a similar API to `append`.

That is,
```golang
func appendTdnfArgsForToolsChroot(tdnfArgs []string) []string {
```